wordpress-mobile/WordPress-Android
# Issue/13326 threat detail

Parent issue #13326

This PR updates ThreatDetail screen
- Displays Fixed header next to the icon for Fixed threats
- Displays Found header below the icon for Fixed threats
- Displays Found header next to the icon for non-fixed threats (current, ignored)
- Update icons background based on threat status (red for current, grey for ignored, green for fixed)

To test:
1. Make sure Scan feature flag is enabled
2. Open threat detail for Current, Fixed and Ignored threats and make sure it matches the behavior described above


Notes:
1. `threat_details_list_header.xml ` is using styles from Jetpack styles (eg. Jetpack.Header). I'm not sure if that's fine, wdyt?
2. Updating the tests was a bit tricky - I think the existing tests are trying to test too much within one method. We might want to consider splitting them into multiple smaller tests where each test verifies only a "single" behavior.  
 
PR submission checklist:
